packages/headings/src/sass/_module.scss

```diff
@@ -26,7 +26,7 @@
 	.uikit-display-#{$i} {
 		font-family: $uikit-font;
 		display: block;
-		line-height: $uikit-leading; //1.142857143;//1.4 from 16 (default: core)
+		line-height: ($uikit-leading * 0.875); //1.4 from 16 (default: core)
```

Comment:
no that's just yet another var to keep in your mental model. No we need a better way of converting them. Ideally we should have something like: `leading(1)` `leading(2)` `leading(3)` `leading(4)` etc... or we have a mixin that takes the font-size and calculates the line height from there. We can curate the line height in there too if we feel certain font sizes should have different edge cases.